src/platform/datapath_epoll.c

```diff
@@ -189,155 +167,10 @@ typedef struct CXPLAT_RECV_MSG_CONTROL_BUFFER {
 
 } CXPLAT_RECV_MSG_CONTROL_BUFFER;
 
-#ifdef DEBUG
-#define CXPLAT_DBG_ASSERT_CMSG(CMsg, type) \
-    if (CMsg->cmsg_len < CMSG_LEN(sizeof(type))) { \
-        printf("%u: cmsg[%u:%u] len (%u) < exp_len (%u)\n", \
```

Comment:
I never noticed we had a printf assert. We should replace this with a real assert.